examples/plotting_data.py

```diff
@@ -14,24 +14,38 @@
 fig, ax = plt.subplots(figsize=(12, 5))
 ax, data = plots.eng_format(ax, data, "W", 1000)
 
-start_date = "2019-12-01 00:00:00"
-end_date = "2019-12-13 23:00:00"
-plots.plot_dispatch(
-    ax=ax,
-    df=data,
-    bus_name="A-electricity",
+start_date = "2016-01-01 00:00:00"
+end_date = "2016-01-10 23:00:00"
+# filter timeseries
+df = data.copy()
```

Comment:
Why do you need this copy() here?

Reply:
I thought it would be necessary since the loaded data could be used to plot different time intervalls. But I checked it again and it is not necessary to make a copy. I removed it.

Reply:
It is needed to keep the original data untouched. I think it can be avoided by performing the preparation of the data only once in the beginning and then calling the plotting function. Currently the prepare_dispatch_data() function is called before plot_dispatch() and before plot_dispatch_plotly() again.
Should I change it to one block of preparing of data and then calling the plotting functions?

Reply:
Then, we could also do the conversion of the data only once in the beginning and could make ```eng_format``` to only do just one thing as suggested in https://github.com/rl-institut/oemoflex/pull/25#discussion_r668047269 and move it into plot_dispatch()

Reply:
Yes, this is what I thought. There is no need to prepare it twice except for the different handling of the Engineering Format, right?

Reply:
Yes, exactly. So then I move the preparation to the beginning and the Engineering Formatter into dispatch_plot(), right?

Reply:
Yes.